pkg/middleend/qos_test.go

```diff
@@ -947,20 +947,20 @@ func TestMiddleEnd_GetQosVolume(t *testing.T) {
 	}
 }
 
-func TestMiddleEnd_QosVolumeStats(t *testing.T) {
+func TestMiddleEnd_StatsQosVolume(t *testing.T) {
 	tests := map[string]struct {
 		in      *_go.ObjectKey
-		out     *pb.QosVolumeStatsResponse
+		out     *pb.StatsQosVolumeResponse
 		spdk    []string
 		errCode codes.Code
 		errMsg  string
 	}{
 		"empty QoS volume id is not allowed ": {
```

Comment:
empty QoS **volume name** is not allowed

Reply:
renamed